lib/src/connection.dart

```diff
@@ -40,8 +40,8 @@ class PostgreSQLConnection extends Object
   /// [useSSL] when true, uses a secure socket when connecting to a PostgreSQL database.
   PostgreSQLConnection(
     this.host,
-    this.port,
     this.databaseName, {
+    this.port = 5432,
```

Comment:
I'd like to have this change in a different PR. Could you please revert this part, and do it separately after this gets merged? It would be even better if we could first release this a null-safe version, and then do the breaking change in a separate release, just to minimize the risk of doing too many things at once.

Reply:
I had the same thought as I pushed, but I figured I might as well get an opinion on it. Will revert.